docs/modules/plugins/partials/proxy-properties.adoc

```diff
@@ -21,3 +21,8 @@
 |host.docker.internal
 |empty
 |Overrides the proxy host which will be passed to browser
+
+|proxy.har.on-failure
+|true/false
+|false
+|Provides possibility to attach har file for failed steps
```

Comment:
we need to emphasize, that har will be attached to each failed step in this mode